internal/common/db.go

```diff
@@ -23,4 +23,7 @@ type Cluster struct {
 
 	// The lease acquired for API vip
 	IngressVipLease string `gorm:"type:text"`
+
+	// The ID of the subscription created in AMS
+	AmsSubscriptionID string `json:"ams_subscription_id"`
```

Comment:
Didn't we talked about making it public so UI can use it as well? 
@avishayt what do you think?

Reply:
Since the UI are accessing all our cluster using the clusterID and they get the clsuterID from the `cluster_id` field of AMS subscription then the conclusion was that there is no case to make it public.